pyccel/errors/errors.py

```diff
@@ -304,7 +304,7 @@ def __str__(self):
         return text
 
 if __name__ == '__main__':
-    from pyccel.parser.messages import NO_RETURN_VALUE_EXPECTED
+    from pyccel.errors.messages import NO_RETURN_VALUE_EXPECTED
     from pyccel.parser.messages import INCOMPATIBLE_RETURN_VALUE_TYPE
     from pyccel.parser.messages import UNDEFINED_VARIABLE
```

Comment:
Should we get all error messaged from `pyccel.errors` now?

Reply:
Good spot. Not sure how I managed to only correct 1 of 3 imports